src/shared/ui/Loader/index.ts

```diff
@@ -0,0 +1,2 @@
+import { Loader, LoaderSize } from "./Loader";
+export { Loader, LoaderSize };
```

Comment:
можно и короче так:
```suggestion
export { Loader, LoaderSize } from "./Loader";
```

некрит